bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/service/HeartbeatService.java

```diff
@@ -27,7 +27,7 @@
  */
 public class HeartbeatService implements HttpEndpointService {
 
-    public static final String HEARTBEAT = "OK\n";
+    public static final String HEARTBEAT = "OK" + System.lineSeparator();
```

Comment:
This is not good, because here we are in the HTTP API, the result cannot depend on the OS that is running the bookie